Sources/SwiftWin32/Support/WindowMessage.swift

```diff
@@ -0,0 +1,1025 @@
+/**
+ * Copyright © 2021 Ryan Allan <rjallan@g.ucla.edu>
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ **/
+
+import WinSDK
+
+struct WindowMessage {
+  static func debugDescription(for uint: UINT) -> String {
+    if let message = messages[Int(uint)] {
+      return message
+    }
+    return "Unknown Message"
+  }
+
+  static let messages : [Int : String] = {
+    var messages : [Int : String] = [:]
+    var keys = Set<Int>()
+    for (key, value) in keyValues {
+      if !keys.contains(key) {
+        keys.insert(key)
+        messages[key] = value
+      }
+    }
+    return messages
```

Comment:
This seems more complicated than necessary.

```suggestion
    return Dictionary<Int, String>(keyValues, uniquingKeysWith: { (first, _) in first })
```